manage_breast_screening/participants/views.py

```diff
@@ -0,0 +1,47 @@
+from logging import getLogger
+
+from django.http import Http404
+from django.shortcuts import get_object_or_404, render
+from django.urls import reverse
+
+from .models import Appointment, Participant
+from .presenters import ParticipantPresenter
+
+logger = getLogger(__name__)
+
+
+def get_back_link(request):
```

Comment:
Given we're going to have to reimplement this would it be better to just redirect back to the referrer (or not include this at all) for now to save passing the appointment_id around which we probably won't want to do in the future?

Reply:
Redirecting to the referrer isn't something I'd want to do in the future either. It seems brittle to assume the referrer is always going to be an appointment page.

From a security perspective I believe the referer header also shouldn't be trusted as it can be manipulated by the client. I'm not sure how an attacker would actually exploit this but I wouldn't feel comfortable leaving it in without validation.

Would you prefer to just take out the back link?